tests/helics/CMakeLists.txt

```diff
@@ -35,7 +35,8 @@ endif()
 # Tests for other CMake projects including and using HELICS
 add_test(find-package-tests ${CMAKE_CTEST_COMMAND} -C ${CMAKE_BUILD_TYPE}
                             --build-and-test "${CMAKE_CURRENT_SOURCE_DIR}/find_package_tests" "${CMAKE_CURRENT_BINARY_DIR}/find_package_tests"
-                            --build-generator ${CMAKE_GENERATOR}
+                            --build-generator "${CMAKE_GENERATOR}"
+                            --build-generator-platform "${CMAKE_GENERATOR_PLATFORM}"
```

Comment:
are we running these tests on Appveyor?

Reply:
They are running on AppVeyor.

Reply:
is --build-generator-platform a valid command if we are not on cmake 3.14?

Reply:
They are running on Appveyor.

Reply:
Well --build-generator-platform shows up in the docs for previous versions of cmake, as does the CMAKE_GENERATOR_PLATFORM variable. If nothing else, it seems to be a no-op when not supported.